app/views/people/_incumbency.html.haml

```diff
@@ -0,0 +1,11 @@
+%li.list__item
+  - if incumbency.house.nil?
+    %h2= link_to(incumbency.seat.house.name, house_path(incumbency.seat.house.graph_id))
+  - else
+    %h2= link_to(incumbency.house.name, house_path(incumbency.house.graph_id))
+
+  %ol
+    %li
+      = "#{l(incumbency.start_date, format: :default)}" unless incumbency.start_date.nil?
+      = incumbency.end_date.nil? ? " to Present" : " to #{l(incumbency.end_date, format: :default)}"
```

Comment:
Could I get some clarification on this?  If this was a decorator, it would still return true or false, so would be used in the same way as .nil?.  Would the purpose of doing this then, just for it to be a bit clearer in the views?  Also, would it be incumbency.present?  I'm not sure if this would be clear that it was checking for the presence of an end_date.  If it is meant to be incumbency.end_date.present?  then I can't currently decorate an end_date as it's not a Grom::Node.  Hope that made sense, just thinking through this one and I was a bit unclear about what you meant.